### ISSUE TITLE
Feature request: Confirm (Ask Bing Chat) button for Bing Chat queries.

### ISSUE THREAD

**Issue by bigplayer-ai** [AI-Anywhere/AI-Anywhere#84]
Feature Request: Add a button to confirm asking Bing Chat
Hello,
I noticed that there is a limit for how many messages you can send to Bing Chat each day. I would like to request a feature that adds a button to confirm asking Bing Chat before sending the request. This way, I can avoid wasting my daily quota on accidental or unwanted queries.
Example for, Ask Bing Chat button:
![Example for, Ask Bing Chat button](https://github.com/haozi/New-Bing-Anywhere/assets/119108387/46f32eac-6ce4-413f-8ee8-fa86d8b272b1)

**Comment by haozi** [comment]
The manual mode was designed a long time ago, but there is no UI interface. First, this plugin will cache queries to save tokens. Second, most people cannot reach the daily usage limit, it will affect the interactive experience.

**Comment by bigplayer-ai** [comment]
Yes, but sometimes for privacy’s sake, I don’t want the search query to reach Microsoft servers. With this feature, we will have better privacy since we can choose which query to send and which query not.
